guides/users/users/overview.md

```diff
@@ -0,0 +1,78 @@
+# Overview
+
+Solidus includes a straightforward user management system. Users are your stores
+customers, as well as your store administrators who need access to the Solidus
+admin interface.
+
+You can search and sort users using a few types of information:
+
+- **Email**: Users access your store and make orders using an identifying email
+  address.
+- **Roles**: Users may have different roles (or permissions) depending on how
+  much access you want them to have to your store. By default, users can have
+  the role `admin` or no role. See [Roles](#roles) for more information.
+- **# Orders**: This field lists the number of orders a customer has made since
+  their account was created.
+- **Total Spent**: This field lists the amount of money that the customer has
+  spent on all of their orders.
+- **Member Since**: The date that the user's account was created.
+
+## User information 
+
+Solidus stores additional information about users. This information relates the
+users to other aspects of your store like payments and orders.
+
+The following information is tracked against your users:
+
+- **Addresses**: Customers have at least one address on file. This address can
```

Comment:
Maybe explain that every time a user enters a new address we associate the address with the users account.

Reply:
Thanks for drawing attention to this, I will tweak this sentence.

But I think that this is expected behavior and doesn't need to be explained.

If a customer enters a new address in their second order, then the new address "updates" the address on file that is exposed in the backend UI.

And at the moment, the backend UI doesn't expose the list of old or alternative addresses.